src/app/portlets/dot-edit-page/content/components/dot-form-selector/dot-form-selector.component.ts

```diff
@@ -39,8 +56,17 @@ export class DotFormSelectorComponent implements OnInit {
                 this.messages = messages;
             });
 
+        this.paginatorService.paginationPerPage = 5;
         this.paginatorService.url = 'v1/contenttype?type=FORM';
     }
+    ngOnChanges(changes: SimpleChanges) {
+        setTimeout(() => {
+            if (changes.show.currentValue === true) {
+                this.contentMinHeight =
+                    this.dialog.dialog.nativeElement.getBoundingClientRect().height - 100;
```

Comment:
If you get the `height` from the `.ui-datatable` you don't need to substract anything and the problem with substract is that we change something in the header, then the math will break.

```suggestion
                    this.dialog.dialog.nativeElement.querySelector('.ui-datatable').getBoundingClientRect().height;
```